src/index.js

```diff
@@ -1,5 +1,6 @@
 import './css/style.css';
 import ToDoCRUD from './modules/todoCRUD.js';
+// import { saveData, loadData } from './modules/localStorage.js';
```

Comment:
Kindly remove unused codes so as to have a clean codebase that is easily maintainable 😄